src/styles/main.scss

```diff
@@ -0,0 +1,44 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stopwatch {
+  display: flex;
+  position: absolute;
+  justify-content: center;
+  width: 80vmin;
+  height: 80vmin;
+  border-style: dotted;
+  border-radius: 50%;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+
+  &__min {
```

Comment:
looks like our styles for `min` and `sec` are common
let's combine them and create a modifier with styles that need to be overridden